kong/provider.go

```diff
@@ -97,5 +110,10 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		AdminToken:         d.Get("kong_admin_token").(string),
 	}
 
-	return gokong.NewClient(config), nil
+	config := &config{
+		adminClient:   gokong.NewClient(kongConfig),
+		strictPlugins: d.Get("strict_plugins_match").(bool),
```

Comment:
not sure on the value of being able to set this globally?  Wnat does this give you over just letting the user set it on each plugin?

Reply:
In case of my setup, I want to always be user that all plugins have strict matching enabled. I don't want to check every time if all plugins have this flag set. Of course, if the user want to they are still able to set the flag for each plugin separately: https://github.com/kevholditch/terraform-provider-kong/pull/87/files#diff-9362b3b5952709e608ac6706b0949606R59

Reply:
Ok makes sense, can you please check the build as it is failing on this branch, you can bump the env vars to get it to run higher versions of kong

Reply:
I've upgraded Kong version from `1.0.0` to `1.4.2`. It works now

Reply:
BTW. I've also fixed goreleaser configuration